content/water-data-science-2021.md

```diff
@@ -0,0 +1,94 @@
+
+Ok...enough of my rambling and onto the fun stuff.
+
+A renewed focus on water data visualizations {#vizlab}
+--------------------
+
+![USGS VIZLAB BANNER](/static/FY21DataSci/vizlab_banner.png)
+
+In 2021 we are stepping up our data visualization game. Step one was hiring Colleen and Ellen to join the team as full-time visualization specialists, as well as Hayley, who does viz, modeling, and workflows. Next, we're holding ourselves to the goal of greater output of high-quality data visualizations. We first started experimenting with data visualization in 2014, visualizing the drought conditions in California with a mix of tech that included parallax and D3 and started using the banner of USGS "Vizlab" (visualization laboratory), going on to release a number of data visualizations on water topics including [microplastics in our waterways](https://labs.waterdata.usgs.gov/visualizations/microplastics/), [climate change and freshwater fish](https://labs.waterdata.usgs.gov/visualizations/climate-change-walleye-bass/index.html), and [U.S. water use](https://labs.waterdata.usgs.gov/visualizations/water-use-15). We also have tested different ways to communicate the complexity of flood timing during hurricane events, using both [interactive](https://labs.waterdata.usgs.gov/visualizations/hurricane-maria/) and [GIFs/videos](https://prd-wret.s3.us-west-2.amazonaws.com/assets/palladium/production/s3fs-public/thumbnails/image/Sally_2020_24fps.gif) that were rendered shortly after landfall. But we’re making a major leap this year with the way we produce visualizations and we’ve upped our standards for design. Check out [water sci and mgmt in the Delware](https://labs.waterdata.usgs.gov/visualizations/delaware-basin-story/index.html#/), [gages through the ages](https://labs.waterdata.usgs.gov/visualizations/gages-through-the-ages/index.html#/), and [fire hydro](https://labs.waterdata.usgs.gov/visualizations/fire-hydro/index.html#/) for a view into early progress. We’re even working on a dataviz that explains how process-guided deep learning models make more accurate predictions. I’m excited to see what’s next. 
+
+
+Advancing machine learning architectures for water prediction {#pgdl}
+--------------------
+
+<div class="grid-row">
+{{< figure src="/static/FY21DataSci/pgdl_flubber_small.gif" alt="Process-guided deep learning predictions of stream temperature" class="side-by-side" >}}
```

Comment:
alt text needs to be actually descriptive.  What is going on in this short video? [how to describe an animal video](https://veroniiiica.com/2020/04/04/video-descriptions-for-animal-videos/)

There should also be a caption for the image (using `caption=`) which can be markdown.